README.md

```diff
@@ -1,141 +1,133 @@
-    - [ ] Plugin adds all items from the collection to the index
+(This plugin may work with the older Strapi versions, but these are not tested nor officially supported at this time.)
 
--> This create the interraction between strapi and MeiliSearch
+**Node / NPM versions**:
 
-### C
+- NodeJS >= 12.10 <= 14
+- NPM >= 6.x
 
-- [ ] Possibility to add more than one collection (hear multiple indexes)
-- [ ] Add a listener on changes on all collections (ex if a restaurant is deleted it should be deleted from meilisearch as well)
+**We recommend always using the latest version of Strapi to start your new projects**.
 
--> This create all basic needs of a user with MeiliSearch
+## 🌎 Community support
 
-### E (BONUS or open for debate)
+- For general help using Strapi, please refer to [the official Strapi documentation](https://strapi.io/documentation/).
+- Strapi Slack [channel](https://slack.strapi.io/)
 
-- [ ] Add customizable collections Relations.
-    Exemple: Two collections: Restaurants and Category. Category has a many to many relation with Restaurant. Automatic array creation should be created when indexing both. Example a Restaurant with its relation to Category
-```js
-{
-    id: '1'
-    name: 'La belle frite',
-    Category: ['Gras', 'Belge', 'Gastronomique']
-}
+## Plugin's goal
 
-```
-And without
-```js
-{
-    id: '1'
-    name: 'La belle frite'
-}
+We want to create a Plugin that provides automatic actions and an interface on the strapi dashboard that helps the user start with MeiliSearch. The User should be able to create or remove new collections from MeiliSearch.
 
-```
-## Consideration
+In the background, the plugin should re-index the documents that have been changed in the collections.
+For exemple, If I change the name of the restaurant `Tonio` with `Tony` in a strapi collection, an update should automaticly be done to update it aswell on MeiliSearch.
 
-- [ ] adding a preview searchbar in the plugin page
+No additional routes should be created on strapi for the front end users as they will be using the MeiliSearch API to search and not Strapi (unless we want to make this possible?).
 
-## TODO
-- [ ] Steps on issues and notions
-- [ ] How do they handle testing ? 
+## MVP
 
+- Downloadable plugin
+- Communication with a running MeiliSearch
+- Interface with the following:
+    - Add host and keys
+    - Ability to chose which collections should be indexed in MeiliSearch
+    - Every time a collection is updated, the updated element should be updated in MeiliSearch as well (listener to add, edit and delete)
+    - Possibility to remove a collection from MeiliSearch
 
-### Not in MVP
+### Bonus
 
-- [ ] Settings on each index in a JSON string
-- [ ] Warning on re-indexation when changing Settings ? (find a way)
-- [ ] more friendly settings UI (still ugly but not a json string) attached to each index. 
+- Possibility for the user to click `add to meilisearch` on Collection creation
```

Comment:
Don't detail it here, it means maintaining it if we change our minds, redirect to the issues :) Issues will be up to date.